src/modules/roc_netio/target_posix/roc_netio/sendto.h

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2020 Roc authors
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+//! @file roc_netio/target_posix/roc_netio/sendto.h
+//! @brief Wrapper for non-blocking sendto() syscall.
+
+#ifndef ROC_NETIO_SENDTO_H_
+#define ROC_NETIO_SENDTO_H_
+
+#include "roc_address/socket_addr.h"
+#include "roc_core/iallocator.h"
+#include "roc_core/list_node.h"
+#include "roc_core/refcnt.h"
```

Comment:
The last three includes are not needed here. What we need is `roc_core/stddefs.h` for `size_t`.